docs/how-tos/image-data.md

```diff
@@ -93,7 +99,8 @@ You can also [export your data from the Viam app](/services/data/export/) to a d
 You now know how to capture and sync image data.
 
 {{< alert title="Avoid fees by disabling data capture" color="note" >}}
-If this is not a production machine and you have confirmed that your machine is capturing and syncing data, disable data capture in the camera configuration to avoid [incurring fees](https://www.viam.com/product/pricing) for capturing large amounts of test data.
+If this is not a production machine and you have confirmed that your machine is capturing and syncing data, you should now disable data capture in the camera configuration to avoid [incurring fees](https://www.viam.com/product/pricing) for capturing large amounts of test data.
```

Comment:
it might be good to make that an explicit step like on the sensor how-to. That make it less easy to miss it I think